tools/cldr-apps/js/src/esm/cldrRef.mjs

```diff
@@ -0,0 +1,13 @@
+import { ref } from "vue";
+
+/**
+ * The current user locale. Read only.
+ */
+const locale = ref(null);
+
+/**
+ * The current XPath (hex id). Read only.
+ */
+const xpath = ref(null);
+
+export { locale, xpath };
```

Comment:
I don't see any need/benefit for this whole mechanism

anyway, if we had to do it, getters/setters would be far preferable to exporting and importing variables